src/chrome/content/rules/HangSengBank.xml

```diff
@@ -14,17 +14,18 @@
 
 	<!--
 	Exceptions: 
-	- HSBCnet logon page
+	- CMS (404 Not Found)
 	-->
-	<exclusion pattern="^http://www\.hangseng\.com\/cms\/sidnet\/files\/hsbcnet" />
+	<exclusion pattern="^http://www\.hangseng\.com\/cms\/" />
 
 	<rule from="^http://hangseng\.com/"
-		to="https://www.hangseng.com/" />
+		to="https://bank.hangseng.com/" />
```

Comment:
The redirect from www to bank works fine on the servers side, so we should not touch it.